autoload/neomake.vim

```diff
@@ -132,7 +132,9 @@ function! neomake#GetMaker(name, ...) abort
             let maker[key] = defaults[key]
         endif
     endfor
-    let maker.name = a:name
+    if !has_key(maker, 'name')
```

Comment:
This bit should go right after line 116, so that `maker.name` definitely exists before we try to use it.